cql3/functions/aggregate_fcts.cc

```diff
@@ -931,88 +649,97 @@ aggregate_fcts::make_count_rows_function() {
 
 shared_ptr<aggregate_function>
 aggregate_fcts::make_max_dynamic_function(data_type io_type) {
```

Comment:
Maybe the word "dynamic" should now be removed from this code, since it's no longer opposite to anything "static" and it will not really be clear to reader what it is trying to say.

Reply:
ok